.travis.yml

```diff
@@ -2,6 +2,7 @@ branches:
   only:
     - master
     - develop
+    - feature_msvc
```

Comment:
```suggestion
```

Reply:
I am accepting the suggested change without understanding it.

Reply:
you shouldn't need to explicitely specify the branch here, testing should run automatically on pull requests